README.md

```diff
@@ -37,6 +39,40 @@ You can use this [shell script][bootstrap-linux] to turn your laptop into an awe
 
 TODO
 
+## Run in Docker
+
+If you don't like to bootstrap your machine you can also run a [Docker][docker] container for development by invoking a single command only. The container will be pulled from our registry. You need to have [Docker Engine][docker-engine] installed first.
+
+~~~ bash
+# [pull and] run container with default rake task (eq. build `_site` and serve via jekyll)
+$ docker run --rm --volume=$(pwd):/usr/src/epages-docs -p 127.0.0.1:4000:4000 -it docker.epages.com/epages/docs-ruby
+~~~
+
+#### Additional commands
+
+~~~ bash
+# [pull and] run container via docker machine on Mac OS X
+$ export IP=$(docker-machine ip `docker-machine active`)
+$ echo "Open in browser: http://${IP}:4000/"
+$ docker run --rm --volume=$(pwd):/usr/src/epages-docs -p ${IP}:4000:4000 -it docker.epages.com/epages/docs-ruby
+
+# Pll and run epages-docs without local repo with current state of upstream branch
+$ docker run --rm -p 127.0.0.1:4000:4000 -it docker.epages.com/epages/docs-ruby:<BRANCH_NAME>
+
+# build image locally
+$ docker build -t docker.epages.com/epages/docs-ruby -f Dockerfile.ruby .
+
+# pull image from registry
+docker pull docker.epages.com/epages/docs-ruby
+~~~
```

Comment:
add tip for building _site on your host: 
docker run --rm --volume=$(pwd):/usr/src/epages-docs -p 4000:4000 -it docker.epages.com/epages/docs-ruby build
or
docker run --rm --volume=$(pwd):/usr/src/epages-docs -p 4000:4000 -it docker.epages.com/epages/docs-ruby rake build